pkg/artifactory/resource_artifactory_remote_repository_test.go

```diff
@@ -70,28 +70,32 @@ func TestAccRemoteDockerRepository(t *testing.T) {
 		"enable_token_authentication":    true,
 		"block_pushing_schema1":          true,
 		"external_dependencies_patterns": []interface{}{"**/hub.docker.io/**", "**/bintray.jfrog.io/**"},
+		"missed_cache_period_seconds":    1800, // https://github.com/jfrog/terraform-provider-artifactory/issues/225
```

Comment:
It'd be great to make branches like `GH-225-fix-missing-cache-period` and then in the comment have `GH-225 - Some comments` that's what I do. Then it makes it easy to track through history

Reply:
👍  I'll close this PR and create another one.